lib/Mojo/File.pm

```diff
@@ -281,8 +281,10 @@ Include hidden files and directories.
 =head2 make_path
 
   $path = $path->make_path;
+  $path = $path->make_path({mode => 0711});
 
-Create the directories if they don't already exist with L<File::Path>.
+Create the directories if they don't already exist with L<File::Path>. Any
+additional arguments are passed through unmodified to L<File::Path::make_path>.
```

Comment:
The `L<File::Path::make_path>` doesn't look right to me, does that actually lead anywhere?